### PR TITLE
Migrate AutomaticGeocoding to ActiveRecord

### PR DESCRIPTION
**Pull request by gonzaloriestra**

### CODE REVIEW

File: app/models/carto/automatic_geocoding.rb [hunk 1]
```diff
@@ -1,9 +1,50 @@
...
+      raise(e)
+    end
+    update_attributes(retried_times: retried_times.to_i + 1, state: 'idle')
+  end
+
+  private
+
+  def set_initial_fields
+    self.created_at = Time.current
+    self.ran_at = Time.current
```
Comment (alrocar) [inline app/models/carto/automatic_geocoding.rb:43]: I've seen `Time.current` being used way less than `Time.now` in the project... not sure why nor the implication of one or the other.

Comment (gonzaloriestra) [reply app/models/carto/automatic_geocoding.rb:43]: It's about time zones. 
- `Time.current` is the same as `Time.zone.now`, which uses the Rails time zone (UTC by default), and it's the recommended way. 
- `Time.now` gets the system time zone, which in our case is also UTC (I hope for all the servers :sweat_smile:).

**Comment by manmorjim** [comment]
Before merge this I think we should check whether the `AutomaticGeocoding` class is being used yet. 

I haven't found any instantiation in the code and the table in prod is empty:

```sql
cartodb_production=# select count(*) from automatic_geocodings;
 count 
-------
     0
(1 row)
```

If this table is not being used anymore then I'd drop it rather than merge this.

**Comment by gonzaloriestra** [comment]
@manmorjim OMG good catch! It looks like I didn't review the usage properly... :man_facepalming: 

I'll check it and remove it if it's really unused.

**Comment by amiedes** [comment]
Should we move this back to Development column?

**Comment by alrocar** [comment]
I'd merge this (we get rid of sequel) and in any case if it's unused we'll 🔥 it eventually.

**Pull request merged**